### ISSUE TITLE
MSE 'bug' in Autoencoder section

### ISSUE THREAD

**Issue by ncullen93** [ageron/handson-ml#2]
In handson-ml/15_autoencoders.ipynb, cell 7 (and chapter 15 in the book), you say an AE is equivalent to PCA w/ an MSE loss, but you have the following code:

reconstruction_loss = tf.reduce_sum(tf.square(outputs - X))  # MSE

This seems to me as the SSE. Probably this is a slight error, or I am just confused. Just a heads up :) Amazing book by the way!

**Comment by ageron** [comment]
Thanks a lot for the heads up @ncullen93. I'll look into this asap.

**Comment by ncullen93** [comment]
No worries.. there are a few very small errors like that, but otherwise this is such an amazing book + code. You're doing great work!

**Comment by ageron** [comment]
You're right, it was confusing, I fixed it (replaced `reduce_sum()` by `reduce_mean()`).  Note that minimizing the MSE and the SSE leads to the same model parameters, so the resulting model is the same.
Thanks for your feedback!
